package.json

```
@@ -22,7 +22,7 @@
     "generate:ts-from-yaml:api": "shx mkdir -p src/generated/payment_manager && shx mkdir -p src/generated/pagopa_proxy && rimraf src/generated/payment_manager/* && rimraf src/generated/pagopa_proxy/* && gen-api-models --strict 0 --api-spec api_payment_manager.yaml --out-dir src/generated/payment_manager --request-types && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v1.2.0/api-spec/api-pagopa-proxy.yaml --out-dir src/generated/pagopa_proxy --request-types",
     "generate:ts-from-yaml:pgs": "shx rm -rf src/generated/pgs && shx mkdir -p src/generated/pgs && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/payment_transactions_gateway_api/pgs-fe/v1/_openapi.json.tpl --no-strict --out-dir src/generated/pgs",
     "generate:ts-from-yaml:ecommerce": "shx rm -rf src/generated/ecommerce && shx mkdir -p src/generated/ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/56eb83e5d10246f55f319af59efde95147934053/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce",
-    "generate:ts-from-yaml:ecommerce-v2": "shx rm -rf src/generated/ecommerce-v2 && shx mkdir -p src/generated/ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/56eb83e5d10246f55f319af59efde95147934053/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce-v2"
+    "generate:ts-from-yaml:ecommerce-v2": "shx rm -rf src/generated/ecommerce-v2 && shx mkdir -p src/generated/ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-2798-openapi-multi-fee/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce-v2"
```

```suggestion
    "generate:ts-from-yaml:ecommerce-v2": "shx rm -rf src/generated/ecommerce-v2 && shx mkdir -p src/generated/ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce-v2"
```
TODO: to be replaced after dependent infra pr merge